### PR TITLE
Do not emit extern enum values for externs.

### PR DESCRIPTION
**Pull request by rkirov**

### CODE REVIEW

File: src/main/java/com/google/javascript/clutz/DeclarationGenerator.java [hunk 1]
```diff
@@ -43,6 +43,7 @@
 import com.google.javascript.rhino.jstype.UnionType;
 import com.google.javascript.rhino.jstype.Visitor;
 
+import com.sun.codemodel.internal.JStatement;
```
Comment (martine) [inline src/main/java/com/google/javascript/clutz/DeclarationGenerator.java]: Is this needed?

File: src/test/java/com/google/javascript/clutz/extern_enum.externs.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+var ns = {};
+
+/** @enum */
+ns.Enum = {A: 0}
```
Comment (martine) [inline src/test/java/com/google/javascript/clutz/extern_enum.externs.js]: It seems you still emitted a value here... ?

Comment (rkirov) [inline src/test/java/com/google/javascript/clutz/extern_enum.externs.js]: The goal was not to emit the value separately too. Previously, we emitted:

declare namespace ಠ_ಠ.clutz.ns.Enum {
  var A: number;
}
along what you see in extern_enum.d.ts

**Comment by martine** [comment]
LGTM

**Pull request merged**